pillar/heroku/discussions.sls

```diff
@@ -199,6 +199,7 @@ heroku:
     OCW_LEARNING_COURSE_BUCKET_NAME: open-learning-course-data-{{ env_data.env_name }}
     OCW_NEXT_AWS_STORAGE_BUCKET_NAME: {{ env_data.OCW_NEXT_AWS_STORAGE_BUCKET_NAME }}
     OCW_NEXT_LIVE_BUCKET: {{ env_data.OCW_NEXT_LIVE_BUCKET }}
+    OCW_NEXT_WEBHOOK_KEY: __vault__::secret-{{ business_unit }}/{{ env_data.env_name }}/ocw-next-webhook-key>data>value
```

Comment:
I would change this to ``__vault__:gen_if_missing:secret-{{ business_unit }}{{ env_data.env_name }}/ocw-next-webhook-key>data>value`

Reply:
Missed the point that they have to be the same value between apps. So would change it to `__vault__::secret-{{ business_unit }}/global/update-search-data-webhook-key>data>value`.

I would also probably change the name of the key to be the same in both apps. Would probably make it easier down the line to identify that the value should be the same.